torch/csrc/distributed/c10d/init.cpp

```diff
@@ -3160,9 +3160,20 @@ such as `dist.all_reduce(tensor, async_op=True)`.
         Arguments:
           tensors(List[torch.Tensor]): List of tensors we want to hash.
       )");
-  module.def("_dump_nccl_trace", []() {
-    return py::bytes(::c10d::dump_nccl_trace());
-  });
+  module.def(
+      "_dump_nccl_trace",
```

Comment:
the names are confusing.  they weren't great before (bc pytorch layer is using 'nccl' name)

but now its extra confusing that the api 'dump_nccl_trace' has an option to exclude the 'traceBuffer' which is further defined as the 'nccl trace buffer' in the docstring.

its pretty unclear to the user what the point of calling `dump_nccl_trace` with includeTraceBuffer=False is.